### ISSUE TITLE
Underline text field missing interactive states

### ISSUE THREAD

**Issue by suztocco** [OfficeDev/office-ui-fabric-react#2401]
### Bug Report

- __Package version(s)__: https://officedevcentersite-devx.azurewebsites.net/fabric#/components/textfield 

- __Browser and OS versions__: Mac Chrome and Safari

Are you willing to submit a PR to fix?  No

Requested priority:  Normal

Products/sites affected: Calendar

#### Describe the issue:

The underline text field input control is missing interactive states (hover, focus, etc)

#### Actual behavior:

The underline stays a static color during interaction and focus

#### Expected behavior:

To match design intent with a hover and focus visual state

### If applicable, please provide a codepen repro:

![image](https://user-images.githubusercontent.com/20954039/28985331-e5705ce8-7916-11e7-96b9-7283c2ed2970.png)

**Comment by erichdev** [comment]
@suztocco Are you still experiencing this? I wasn't able to repro in Chrome or Firefox

**Comment by suztocco** [comment]
I am looking at the Fabric site and I do not see the underline changing color on hover, etc. I am looking at Chrome on Mac
